openverse_catalog/dags/providers/provider_api_scripts/jamendo.py

```diff
@@ -94,7 +94,7 @@ def _get_items():
     while should_continue:
         query_params = _get_query_params(offset=offset)
         batch_data = _get_batch_json(query_params=query_params)
-        if isinstance(batch_data, list) and len(batch_data) > 0:
+        if isinstance(batch_data, list) and len(batch_data) > 0 and item_count < LIMIT:
```

Comment:
What is the purpose of this change? This still looks like testing code that was accidentally left in to me. I believe this will force the jamendo script to stop early after loading the first 200 records.

Reply:
It is debug code, and I have no idea how it crept in here. Thank you for catching it, @stacimc !